test/e2e/play_kube_test.go

```diff
@@ -3665,7 +3665,7 @@ o: {{ .Options.o }}`})
 		kube := podmanTest.Podman([]string{"play", "kube", kubeYaml})
 		kube.WaitWithDefaultTimeout()
 		if IsRemote() {
-			Expect(kube).Error()
+			Expect(kube).To(ExitWithError())
```

Comment:
can we check for the actual error code here not just any, https://github.com/containers/podman/issues/18188